conf/openmetadata.yaml

```diff
@@ -151,6 +151,7 @@ migrationConfiguration:
   flywayPath: "./bootstrap/sql/migrations/flyway"
   nativePath: "./bootstrap/sql/migrations/native"
   extensionPath: ""
+  migrationValidationClass: ${MIGRATION_VALIDATION_CLASS:-"org.openmetadata.service.migration.OpenMetadataMigrationValidation"}
```

Comment:
do we need to make this configurable?

Reply:
In Collate the migrations will be different (since we'll have 1.2.1-collate, etc.)

Reply:
shouldn't the migration validation should be same no?
why hardcode this into the code, instead of reading from file system of bootstrap directory and list out the versions? @pmbrull 
otherwise everyone needs to remember to add this to this part of the code again

Reply:
make sense, I'll update this